### PR TITLE
Dont remove special characters in person names

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Hotfix to prevent special characters from being removed in person names.

# Need Review By (Date)
- asap

# Urgency
- high.

# Steps to Test
1. checkout this branch
1. `drush config-import -y`
1. create a person content and use special characters like dash or & in the first or last name
1. save and verify the special characters are kept.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Review by imonroe — APPROVED** [review]
Looks good, tests green.

File: config/sync/auto_entitylabel.settings.node.stanford_person.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 status: 1
 pattern: '[node:su_person_first_name] [node:su_person_last_name]'
-escape: true
+escape: false
```
Comment (jbickar) [inline config/sync/auto_entitylabel.settings.node.stanford_person.yml:3]: What are the security implications of this change?

Comment (pookmish) [reply config/sync/auto_entitylabel.settings.node.stanford_person.yml:3]: No different than putting special characters in the title on other content types. shouldn't be any security issue.

Comment (pookmish) [reply config/sync/auto_entitylabel.settings.node.stanford_person.yml:3]: this is also the same settings we currently have for news content types.

Comment (jbickar) [reply config/sync/auto_entitylabel.settings.node.stanford_person.yml:3]: So Drupal sanitizes it prior to putting it in the DB? Can you add a [Little Bobby Tables](https://xkcd.com/327/) test?

**Comment by jbickar** [comment]
You purposefully titled this PR about removing special characters `Dont remove special characters in person names` to give me the twitches, didn't you?

File: config/sync/auto_entitylabel.settings.node.stanford_person.yml (hunk 1 above)
Comment (sherakama) [reply config/sync/auto_entitylabel.settings.node.stanford_person.yml:3]: Drupal doesn't often sanitize writing to the db, but rather, wants you to sanitize on output.

Comment (jbickar) [reply config/sync/auto_entitylabel.settings.node.stanford_person.yml:3]: That was my recollection, but I did not phrase my question articulately. Is it safe to say that the DB abstraction layer protects against this not being escaped?

**Pull request merged**